cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -66,13 +66,23 @@ set(configure_command
 # Pass C flags through
 set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
 
+# Pass BUILD_SHARED_LIBS (what about RPATH?)
+set(shared_libs "@BUILD_SHARED_LIBS@")
+if (shared_libs)
```

Comment:
With the change I made this works. It seems that BUILD_SHARED_LIBS is not being forwarded correctly.